src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/SafeHandles.cs

```diff
@@ -185,7 +185,7 @@ public static void DeleteKeyContainer(SafeCertContextHandle pCertContext)
                     {
                         CryptAcquireContextFlags flags = (pProvInfo->dwFlags & CryptAcquireContextFlags.CRYPT_MACHINE_KEYSET) | CryptAcquireContextFlags.CRYPT_DELETEKEYSET;
                         IntPtr hProv;
-                        bool success = Interop.cryptoapi.CryptAcquireContext(out hProv, pProvInfo->pwszContainerName, pProvInfo->pwszProvName, pProvInfo->dwProvType, flags);
+                        _ = Interop.cryptoapi.CryptAcquireContext(out hProv, pProvInfo->pwszContainerName, pProvInfo->pwszProvName, pProvInfo->dwProvType, flags);
```

Comment:
Per discussion at #41876, this uses an explicit discard.

Not checking the result of `success` seems to be very similar to another `CryptAquireContext` https://github.com/dotnet/corefx/blob/b8ca29af97b36d35a47ae3ae925ac9e61a11bfc8/src/Common/src/Interop/Windows/Advapi32/SafeProvHandle.cs#L115-L116

that is also deleting a keyset.